atum/src/main/scala/za/co/absa/atum/utils/FileUtils.scala

```diff
@@ -0,0 +1,12 @@
+package za.co.absa.atum.utils
```

Comment:
Well, they could in theory. I thought the separation clearly communicates the different concerns they solve. 🤷 I will think about it.

Reply:
The more I think about this, the less I think it would improve things. So unless there is an obvious upside in this (that I am not seeing at the moment), I'd rather keep it separate.